components/automate-ui/src/app/pages/+compliance/+profile/+profile-details/profile-details.component.scss

```diff
@@ -6,6 +6,20 @@ chef-loading-spinner {
   width: 100px;
 }
 
+chef-page-header {
+  chef-subheading {
+    max-width: 65%;
```

Comment:
I feel max-width is better as a percentage so that it's a bit more flexible to screen widths than a hard pixel number.